README.MD

```diff
@@ -47,31 +47,31 @@ First we need to download these three Applications: [Hackintool](https://github.
+- Open the application and select 'Post Install Root Patch'.
 - Wait for a while and reboot the system after it finishes.
-- Enjoy your E7250 on Sonoma :)
+- Replace `amfi=0x80` with `-amfipassbeta` under `boot-args` inside your config.plist to restore AMFI support.
+- Reboot and enjoy your E7250 on Sonoma :)
 
 ## Intel WiFi
 
 Starting with v2.3, AirportItlwm is included in the EFI but disabled to ensure stability. To enable Intel WiFi, enable it in the config. The EFI will only have stable versions of AirportItlwm. So if you want to experiment with Intel WiFi, you can do so by visiting the [itlwm repo](https://github.com/OpenIntelWireless/itlwm) and downloading the latest alpha build. Stability can be much worse, so use with caution! Also, do not forget to enable IntelBluetoothFirmware, BlueToolFixup and IntelBTPatcher in the config to enable Bluetooth!
 
-If you experience no internet with AirportItlwm, I suggest switching to itlwm and see if it works there.
+If you experience no internet with AirportItlwm, I suggest switching to itlwm + HeliPort and see if it works there.
 
 Since Monterey, the Bluetooth implementation has changed and therefore causes issues such as: Bluetooth will be broken after sleep. As a workaround, typing the following command in Terminal may fix this issue: `sudo killall -9 bluetoothd BlueTool`
 
 If you want Continuity features: Buy a Broadcom card on eBay or your trusted website. You can check the Dortania buyers guide to see which card is the better option: [Link](https://dortania.github.io/Wireless-Buyers-Guide/)
 
 ## OTA from Monterey to Sonoma
 
-Since v5.0 installing Sonoma via OTA should be very easy. Replace your Monterey EFI with Sonoma and boot as usual. Open 'System Preferences' and go to 'Software Update'. Download macOS Sonoma and proceed with the installation as a normal update. After update, run OpenCore Legacy Patcher to fix GPU acceleration. Here's a little [guide](#GPU-acceleration-on-Ventura-and-newer) you can follow. Keep in mind that replacing Monterey EFI with Sonoma drops AirportItlwm support on Monterey! If you want to upgrade to Sonoma over WiFi, replace the Sonoma version of AirportItlwm with a Monterey version. After the update, you need to roll back the currently version of AirportItlwm to add WiFi support on Sonoma. The same applies to Ventura.
+Since v5.0 installing Sonoma via OTA should be very easy. Open 'System Preferences' and go to 'Software Update'. Download macOS Sonoma and let it prepare the update. Do NOT reboot yet, replace the Monterey EFI with the Sonoma version, then proceed with the upgrade as normal. After the upgrade, run OpenCore Legacy Patcher to fix GPU acceleration. Here's a little [guide](#Restoring-GPU-acceleration-under-Sonoma) you can follow. The same applies to Ventura, with the difference being you'll need to replace the AirportItlwm kext with the Ventura variant.
```

Comment:
I don't think you can upgrade from Monterey to Ventura/Sonoma with the Monterey EFI, since all related patches are disabled. The reason why I did it this way is to have an EFI that works like a real stock MacBookPro12,1. For newer versions of macOS, the Sonoma EFI should be used instead.

Thus, the upgrade path should look like this:
- replace Monterey EFI with Sonoma EFI
  - this enables OTA upgrade on Monterey
  - however this drops Wi-Fi support
    - add Monterey AirportItlwm to fix this issue
- update via System Preferences
- Before reboot or after installation, Sonoma AirportItlwm can be added back
- Lastly, the root patches need to be applied